tests/queries/0_stateless/02775_show_columns_mysql_compatibility.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# This script tests the MySQL compatibility of the SHOW COLUMNS command in ClickHouse
+USER="default"
+PASSWORD=""
+HOST="127.0.0.1"
+PORT=9004
+
+# First run the clickhouse test to create the ClickHouse Tables
+
+echo "Drop tables if they exist"
+${CLICKHOUSE_CLIENT} --query "DROP TABLE IF EXISTS tab"
+${CLICKHOUSE_CLIENT} --query "DROP DATABASE IF EXISTS database_123456789abcdef"
```

Comment:
`no-parallel` can be removed and `${CLICKHOUSE_DATABASE}` can be used to get the unique generated database for this test.